tests/acceptance/features/bootstrap/WebDav.php

```diff
@@ -838,32 +832,48 @@ public function userOnMovesFileUsingTheAPI(
 	}
 
 	/**
-	 * @When /^user "([^"]*)" copies (?:file|folder) "([^"]*)" to "([^"]*)" using the WebDAV API$/
-	 *
 	 * @param string $user
 	 * @param string $fileSource
 	 * @param string $fileDestination
 	 *
-	 * @return void
+	 * @return ResponseInterface
 	 */
-	public function userCopiesFileUsingTheAPI(
+	public function copyFile(
 		string $user,
 		string $fileSource,
 		string $fileDestination
-	):void {
+	):ResponseInterface {
 		$user = $this->getActualUsername($user);
 		$headers['Destination'] = $this->destinationHeaderValue(
 			$user,
 			$fileDestination
 		);
-		$this->response = $this->makeDavRequest(
+		return $this->makeDavRequest(
 			$user,
 			"COPY",
 			$fileSource,
 			$headers
 		);
+	}
+
+	/**
+	 * @When /^user "([^"]*)" copies (?:file|folder) "([^"]*)" to "([^"]*)" using the WebDAV API$/
+	 *
+	 * @param string $user
+	 * @param string $fileSource
+	 * @param string $fileDestination
+	 *
+	 * @return void
+	 */
+	public function userCopiesFileUsingTheAPI(
+		string $user,
+		string $fileSource,
+		string $fileDestination
+	):void {
+		$response = $this->copyFile($user, $fileSource, $fileDestination);
+		$this->setResponse($response);
 		$this->setResponseXml(
```

Comment:
I think. these 3 lines should be enough
```suggestion
		$response = $this->copyFile($user, $fileSource, $fileDestination);
		$this->setResponse($response);
		$this->$this->pushToLastHttpStatusCodesArray();
}
```